javalib/src/main/scala/java/util/logging/Level.scala

```diff
@@ -0,0 +1,48 @@
+package java.util.logging
+
+object Level {
+  val OFF: Level     = new Level("OFF", Int.MaxValue)
+  val SEVERE: Level  = new Level("SEVERE", 1000)
+  val WARNING: Level = new Level("WARNING", 900)
+  val INFO: Level    = new Level("INFO", 800)
+  val CONFIG: Level  = new Level("CONFIG", 700)
+  val FINE: Level    = new Level("FINE", 500)
+  val FINER: Level   = new Level("FINER", 400)
+  val FINEST: Level  = new Level("FINEST", 300)
+  val ALL: Level     = new Level("ALL", Int.MinValue)
+
+  private val knownLevels = List(OFF, SEVERE, WARNING, INFO, CONFIG, FINE,
```

Comment:
Consider creating a map instead, and having a trivial implementation for `parse`.
```scala
private lazy val knownLevels = Map[String, Level](
    OFF.name -> OFF, OFF.intValue -> OFF,
    SEVERE.name -> SEVERE, SEVERE.intValue -> SEVERE,
    WARNING.name -> WARNING, WARNING.intValue -> WARNING,
    ...
)
```